central/scannerdefinitions/handler/handler.go

```diff
@@ -122,43 +122,52 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *httpHandler) get(w http.ResponseWriter, r *http.Request) {
-	if fileType := r.URL.Query().Get("file"); fileType != "" {
-		if v4FileName, exists := v4FileMapping[fileType]; exists {
+	uuid := r.URL.Query().Get(`uuid`)
+	fileName := r.URL.Query().Get(`file`)
+
+	// If only file is requested, then this is for Scanner v4.
+	if fileName != "" && uuid == "" {
+		if v4FileName, exists := v4FileMapping[fileName]; exists {
 			h.getV4Files(w, r, mappingUpdaterKey, v4FileName)
 			return
 		}
-		if fileType == "cvss" {
+		if fileName == "cvss" {
 			h.getV4Files(w, r, cvssUpdaterKey, "")
 			return
 		}
 		writeErrorNotFound(w)
+		return
```

Comment:
Added this, as it was possible we should stop handling the request once we know there is an error